### PR TITLE
[RB-219] Create a generator for the config

### PR DESCRIPTION
**Pull request by Gabrielsods**
Create a new initializer in `config/initializers/rubyblok.rb` with the command `rails generate rubyblok:install`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,11 +8,13 @@ gemspec
 gem "rake"
 
 group :development, :test do
+  gem "rails"
```
Comment (ziklodi) [inline Gemfile:11]: why do we need to include rails gem? I thin the generators are included in railties already.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -2,15 +2,18 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "rails"
```
Comment (ziklodi) [inline spec/spec_helper.rb:5]: do we need rails?

File: lib/generators/templates/rubyblok.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Rubyblok.configure do |config|
```
Comment (ziklodi) [inline lib/generators/templates/rubyblok.rb:1]: do we want to add this on the top?
```
# frozen_string_literal: true
```

File: Gemfile (hunk 1 above)
Comment (Gabrielsods) [reply Gemfile:11]: -Yes we need, railties was not enough.

File: spec/spec_helper.rb (hunk 2 above)
Comment (Gabrielsods) [reply spec/spec_helper.rb:5]: ^^ see comment above

File: spec/generators/install_generator_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  context "when executing the install generator" do
+    context "when the file doesn't exist" do
+
+      before do
+        run_generator
+      end
+
+      it "should create the initializer file" do
+        assert_file("config/initializers/rubyblok.rb")
```
Comment (ziklodi) [inline spec/generators/install_generator_spec.rb:21]: can we test the content of the file? ie
```
assert_file("config/initializers/rubyblok.rb") do |content|
  assert_match(/something/, content)
end
```

probably we need to mock constants like
```
stub_const("STORYBLOK_API_TOKEN", "token")
```

**Review by ziklodi — APPROVED** [review]

**Pull request merged**